guides/phoenix_mix_tasks.md

```diff
@@ -30,7 +30,7 @@ mix phx.server         # Starts applications and their servers
 
 We have seen all of these at one point or another in the guides, but having all the information about them in one place seems like a good idea. And here we are.
 
-#### `mix phx.new`
```

Comment:
FYI: when new version of ex_doc is released, `mix phx.new` as well as `mix help phx.new` would auto link to task docs [1] [2] so this can stay as is.

[1] https://github.com/elixir-lang/ex_doc/pull/911
[2] https://github.com/elixir-lang/ex_doc/pull/919

Reply:
I'll update the PR, but keep the change from `####` to `###`